go.mod

```diff
@@ -43,4 +43,4 @@ require (
 
 replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
 
-replace github.com/onsi/ginkgo => github.com/onsi/ginkgo v0.0.0-20190716150225-054541502288
+replace github.com/onsi/ginkgo => github.com/manosnoam/ginkgo v0.0.0-20190902135631-1995eead7451
```

Comment:
di you send them a PR?, can we may be fetch the merged version?

Reply:
Yes, my PR #601 in Ginkgo has passed CI: "A new CLI option: -ginkgo.reportFile <file path>"
https://github.com/onsi/ginkgo/pulls

There's also an old enhancement request, where I suggested my PR there:
https://github.com/onsi/ginkgo/issues/320

It may take a month+ until they review, since the list of PRs is long.

In the meantime, see the rusults of junit.xml files created:

> /go/src/github.com/submariner-io/submariner/pkg/routeagent/controllers/route/junit.xml
> /go/src/github.com/submariner-io/submariner/pkg/util/junit.xml

Reply:
My Ginkgo PR https://github.com/onsi/ginkgo/pull/601 has been merged. We can use --reportFile